custom/ilsgateway/models.py

```diff
@@ -550,7 +561,7 @@ def location_edited_receiver(sender, loc, moved, **kwargs):
         )
 
     group = last_location_group(loc)
-    if group != loc.metadata['group']:
+    if not loc.sql_location.location_type.administrative and group != loc.metadata['group']:
```

Comment:
There is possibility to set up group for non-facility location in application but it doesn't make sense from reports perspective so we can just ignore that.